### ISSUE TITLE
bank: forgotten password

### ISSUE THREAD

**Issue by denisdimitrov** [denisdimitrov/scrum#6]
Users should be able to retrieve their forgotten passwords by entering their email and then their password is sent to their email.

1. Email exists.
2. Email doesn't exist.

**Comment by mgenov** [comment]
The functionality actually is: reset user password 

so each user should be able to reset it's password.

**Comment by denisdimitrov** [comment]
Yes, the password should be reset instead of sending the old one. My bad. It's fixed now.
